docs/keyring-core.md

```diff
@@ -84,15 +84,14 @@ export interface KeyringContextActions {
    */
   setCurrentSpace: (did: DID) => Promise<void>
   /**
-   * Register the current space, verify the email address and store in secure
-   * storage. Use cancelRegisterSpace to abort. Automatically sets the
-   * newly registered space as the current space.
+   * Register the current space and store in secure storage. Automatically
+   * sets the newly registered space as the current space.
    */
   registerSpace: (email: string) => Promise<void>
   /**
    * Abort an ongoing account registration.
    */
-  cancelRegisterSpace: () => void,
+  cancelAuthorize: () => void,
```

Comment:
This would be backward incompatible, but the PR title does not have a `feat!`

Reply:
yeaaaa I think it was a bug in this major release that this wasn't changed, so I'd like it to ship in a minor release. I know it's technically semver heresy, but I also think it was just a bug that this didn't break with the last major

Reply:
put another way - if someone has already gone through the last major version upgrade but is depending on this function, I'd rather have their code break when they update the minor rather than requiring them to do another major version upgrade here, so my preference is to ship this in a minor

Reply:
Would still be helpful to get @alanshaw's brain on this one - @gobengo and I went back and forth but ultimately decided just shipping a new major release is probably the least ugly thing to do here, rather than shipping a minor release with a backwards incompatible change.

Probably a moot point given the current level of outside adoption of these libraries, but worth being on the same page about regardless.